src/observation/subscribeToSimpleQuery/index.js

```diff
@@ -54,8 +54,6 @@ export default function subscribeToSimpleQuery<Record: Model>(
   // observeQueryWithColumns
   alwaysEmit: boolean = false,
 ): Unsubscribe {
-  invariant(!query.hasJoins, 'subscribeToSimpleQuery only supports simple queries!')
```

Comment:
Hmm do we even need `hasJoins` now? I guess it is a part of Query API (although marked as internal) so not sure if we can remove it...

Reply:
Good point, I'll search to see if anything even uses this internally.

Reply:
deprecated in next PR